### ISSUE TITLE
use `attr_reader` to define trivial reader methods on class-level instance variables

### ISSUE THREAD

**Issue by JeanMertz** [bbatsov/rubocop#1412]
This

```ruby
    def self.skipped_authentication
      @skipped_authentication
    end
```

triggers `Use attr_reader to define trivial reader methods.`.

I know it can be written as such:

```ruby
    class << self
      attr_reader :skipped_authentication
    end
```

but that's the same amount of lines and not much clearer in my eyes. I'd say `def self.*` shouldn't trigger the trivial reader methods error.

**Comment by lumeet** [comment]
This was fixed by 3d8417af0c6aa7ca2378139d20987ade3023e860.
